blatann/gatt/gatts.py

```diff
@@ -68,7 +68,7 @@ class GattsCharacteristicProperties(gatt.CharacteristicProperties):
     def __init__(self, read=True, write=False, notify=False, indicate=False, broadcast=False,
                  write_no_response=False, signed_write=False, security_level=gatt.SecurityLevel.OPEN,
                  max_length=20, variable_length=True, sccd=False,
-                 cccd_security_level=gatt.SecurityLevel.OPEN,
+                 cccd_write_security_level=gatt.SecurityLevel.OPEN,
                  user_description: GattsUserDescriptionProperties = None,
                  presentation_format: PresentationFormat = None):

```

Comment:
Sorry one last change I didn't notice before. Could we add the new param to the end of the list as to not break backwards compatibility with positional args?

Reply:
I am not a python expert, I think I tried it, but something did not work out properly. Feel free to modify the patch so that it fits best.